packages/eslint-plugin-template/src/rules/i18n.ts

```diff
@@ -457,12 +436,13 @@ function isAttributeAllowed(
 }
 
 function isBoundTextAllowed(
-  boundTextAllowedPattern: RegExp,
+  allowedBoundTextPattern: RegExp,
   {
     value: {
       ast: { strings },
     },
   }: StronglyTypedBoundText,
 ) {
-  return boundTextAllowedPattern.test(strings.join('').trim());
+  const text = strings.join('').trim();
+  return !PL_PATTERN.test(text) || allowedBoundTextPattern.test(text);
```

Comment:
We only consider it as an allowed text if it doesn't contain letters or if it was deliberately allowed by the user via `boundTextAllowedPattern`.

Reply:
Please can you apply this as a comment to the code too? I think it is great to capture your intention there